### DISCUSSION TITLE
Demo app spring-server returns 404 for base route - /playground

### DISCUSSION THREAD

**Discussion by sithsdo** [ExpediaGroup/graphql-kotlin#1761] [Q&A] [answered]
Hey curious if this is a known issue with running the spring-server example locally, but it appears to not be properly registering an entry route. Everything builds correctly, the app starts with no errors but when navigating to http://localhost:8080/playground it throws a 404. Is there information missing from the readme or is this an actual issue?

Steps to reproduce:

1.  clone spring-server example locally 
2. ./gradlew :spring-server:build
3. ./gradlew :spring-server:bootRun

**Comment by samuelAndalon** [comment]
Which version of graphql-kotlin are you using to clone ? or just using what is on master ?

also see:
https://github.com/ExpediaGroup/graphql-kotlin/issues/1679

**Reply by sithsdo** [reply]
I pull directly from master.

**Reply by samuelAndalon** [reply, marked as answer]
try `/graphiql`

also created https://github.com/ExpediaGroup/graphql-kotlin/pull/1762 to point to graphiql IDE, playground is still available but needs to be manually enabled, more info -> https://opensource.expediagroup.com/graphql-kotlin/docs/server/spring-server/spring-properties/

**Reply by sithsdo** [reply]
okay perfect, yeah that works. thanks for the quick help!
